### DISCUSSION TITLE
interactive "Starter Pack"

### DISCUSSION THREAD

**Discussion by VZozul** [sunflower-land/sunflower-land#2683] [Ideas]
I've been contemplating this idea since the previous referral program. I kept thinking, why are we getting something for someone simply registering? There's multi-accounting, friends, acquaintances who will just sign up and forget. To attract people, we need to offer a little more than that. When they join and play a bit, enticed by the freebies, they might decide to stay, and we don't have to gain anything from it. We gain new players! And they are valuable not only to you but to us as well. That's why we should strive to attract players on a complimentary basis. We'll earn later by selling resources. The previous referral system was flawed; it wouldn't attract new players. But offering incentives... I know I'm late; I spent a long time creating my prototype. Given that Figma isn't particularly designed for this purpose, it was challenging to achieve what I envisioned. Adapting the prototype for both pc and mobile was even more difficult, and if it weren't for the recent updates in Figma, I would have struggled for a long time. The scalability of Figma didn't allow me to convey everything as accurately as I wanted, but with the new features, it has become easier, and I managed to finalize the prototype. When I saw the screenshot of your Starter Pack, my spirits sank. Just imagine, you spend so much time, only to discover that the idea has already been implemented, and in a much simpler way, without all the extras. I liked your interpretation of the Starter Pack and the direction you're thinking in. But what remains for me? Still, I decided to share what I've created. Perhaps in the future, you will create something similar, you might find something you like from what I've presented there, in addition to the interactive 'Starter Pack,' there are a few other elements:

1. **Logo** with an owl.
2. **Boulder**, respawns once a week, requiring 3 Boulder Pickaxe. Resources are randomized. Alternatively, it could be designed like a quest in Downbreaker where you need to water every 24 hours, requiring the Boulder Pickaxe once every 24 hours, and then 3 Boulder Pickaxe within 7 days to fully break the rock.
3. **The Floating Whale**, which searches the sea once a day, similar to a treasure chest on Treasure Island. Considering Halving, it might be relevant.
4. **Scattered tools** that can be found randomly in new World locations. They appear in various nooks and crannies. I believe this will be a pleasant incentive for players to spend more time exploring there.
5. **Seed Pouch** (you'll see more details in the prototype). This could be introduced in the next Treasure Island season. As an option, it could be randomly dropped in new locations. After all, pirates might have passed by and lost them.

You will see and interact with everything in the prototype in Figma.

When a person purchases a Starter Pack, depending on the pack, they receive different NFTs from chests, varying amounts of crops, mushrooms, eggs. They can choose a bear, a flag, and clothing. Everything is interactive; they can even change hairstyles right there.

If someone registers through the referral program, offer a discount. All received NFTs will be locked until certain levels or expansions are achieved.

Basic Starter Pack
Intermediate Starter Pack
Advanced Starter Pack

In the prototype, all values are random, just for demonstration purposes. There is a quest in the prototype, very basic and easy, just for illustration.

I apologize in advance; the prototype is quite heavy, so it may have some glitches. Sometimes things may not open or animation is skipped; it's advisable to reload a couple of times before starting and give it some time to load the animation. For some reason, it doesn't work on Safari mobile, but it opens in other browsers, or even better, through the Figma app or PC.

And finally, a small feedback regarding the current Starter Pack. Make a separate announcement specifically for this news. You must understand that you have 40k people subscribed to you, but not everyone will open the link to Medium and read it. A separate announcement is needed.

Below are 2 screenshots, one is the initial page, the second is to understand what the whole prototype looks like behind the scenes. I tried very hard, so don't be lazy and open the prototype)

https://www.figma.com/proto/af9AFmSwJKEYsRpvRptCxF/Sun?page-id=28%3A97&type=design&node-id=49-3896&viewport=625%2C396%2C0.2&t=smUdfJEbdcQz1KdM-1&scaling=min-zoom&starting-point-node-id=49%3A3896&hotspot-hints=0&mode=design

![SN_pr](https://github.com/sunflower-land/sunflower-land/assets/103659539/1f2713e2-f58b-436b-ada5-5e1b298fa086)
![Starter Pack](https://github.com/sunflower-land/sunflower-land/assets/103659539/b47b6328-8c54-489e-b9ce-675c408f1199)

**Comment by adamhannigan** [comment]
@vzozulia Thanks for the detailed suggestion and figma design.

My understanding is that this is an idea to help onboarding + start pack idea. A player would start on a 'sandbox' island that has resources & mystery gifts ready to collect and interact with. Is that right?

This is definitely more engaging for new players and a nice way to understand the gameplay & what is possible with your island. It will add a bit of complexity since these game mechanics aren't supported yet (boulder, picking up tools etc.)

**Reply by VZozul** [reply]
Yes, that's correct. Initially, I had the idea of a Starter Pack to offer some freebies to attract players because, in my understanding, it's the only way to attract players at this stage of the market. Simply receiving something random isn't as interesting; interactivity introduces players to the game and engages them from the first moments. The randomness of what you receive adds a desire to test your luck, and depending on whether the player buys the Basic, Intermediate, or Advanced pack, they will get different random items. Then, I tried to visualize my idea. Since I don't know how to code, Figma was the only option for me. By the way, I hadn't worked with it before, but it turned out that you can create a whole mini-game there. Thanks to this project, I can now do anything in Figma. Logic can be a bit challenging there since it's not designed for such purposes. I'm currently working on the "Lucky Tree" project, and I'm almost done.

Regarding the Boulder, tool pickups, and Seed Pouch, they're not mandatory; regular nodes and tool rewards could be implemented instead. These ideas came up during the process when I needed to figure out the logic of obtaining seeds and getting an ax to chop down trees. That's how I found a solution. In the end, I liked these ideas, so I decided to combine all the ideas into one. As I mentioned before, you might find something you like from what I presented there. The interactive Starter Pack idea itself can be used as a basis, and it can be redesigned differently.

-The Boulder is a helpful tool for new players, and just like with Dawn flowers, it encourages daily activity to achieve results.
-Scattered tools are great motivators to explore new locations and add interactivity to those areas.
-The Seed Pouch is similar to fertilizer but with random seeds inside. Quests will become much more interesting, plus it's a good reward for those who dig on Treasure Island every day. A person digs and collects Seed Pouches, then uses them in quests. Let's say they can use 10 of them, and only on the 10th one, they'll get the required seeds, allowing them to complete the quest faster than others.
-Floating Whales, I just like them.
